package.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package packagerversion="1.10.5" version="2.0" xmlns="http://pear.php.net/dtd/package-2.0" xmlns:tasks="http://pear.php.net/dtd/tasks-1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0 http://pear.php.net/dtd/tasks-1.0.xsd http://pear.php.net/dtd/package-2.0 http://pear.php.net/dtd/package-2.0.xsd">
+  <name>phalcon</name>
+  <channel>pecl.php.net</channel>
+  <summary>Phalcon is a full stack PHP framework offering low resource consumption and high performance.</summary>
+  <description>
+    Phalcon is an open source full stack framework for PHP, written as a C-extension. Phalcon is optimized for high performance. Its unique architecture allows the framework to always be memory resident, offering its functionality whenever its needed, without expensive file stats and file reads that traditional PHP frameworks employ.</description>
+  <lead>
```

Comment:
Could you add all team members?

Reply:
I think it should match up with the pecl account. Will check this otherwise I will add you guys

Reply:
Need to dismiss this. It should match up with the Pecl username. It's only ment for identifying the package maintainers not the code itself.